native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -2565,28 +2557,28 @@ static int proxy_cluster_post_config(apr_pool_t *p, apr_pool_t *plog, apr_pool_t
     ap_add_version_component(p, MOD_CLUSTER_EXPOSED_VERSION);
 
     if (ap_state_query(AP_SQ_MAIN_STATE) == AP_SQ_MS_CREATE_PRE_CONFIG) {
-        ap_log_error(APLOG_MARK, APLOG_EMERG, 0, s, "No watchdog for %d", getpid());
+        ap_log_error(APLOG_MARK, APLOG_CRIT, 0, s, "No watchdog for %d", getpid());
         return OK;
     }
-    ap_log_error(APLOG_MARK, APLOG_EMERG, 0, s, "watchdog for %d", getpid());
+    ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, s, "watchdog for %d", getpid());
```

Comment:
Oops I though I used NOTICE there.

Reply:
Maybe `APLOG_INFO` might be more suitable than `APLOG_NOTICE`? I put there `NOTICE` for now.

Reply:
Yes please put APLOG_INFO there.